### PR TITLE
fixed the price tag

### PR DESCRIPTION
**Pull request by jw642459986**
Price_tag screenshot

Ignore the buttons for now, I plan to style them next.

### CODE REVIEW

**Comment by jw642459986** [comment]
@Ahmed-Elawad Hey man, can you take a look at this for me? Thanks!

File: client/src/index.jsx [hunk 1]
```diff
@@ -78,17 +79,17 @@ class App extends React.Component {
             <div className="BottomCarousel">
                 <h3 className="Module_Header">Similar Products</h3>
                 <Container>
-                    <Left_Button 
+                    <LeftButton 
                     onClick={() => this.handleLeftClick()}
                     disabled={this.state.first === 0}>
```
Comment (Ahmed-Elawad) [inline client/src/index.jsx:84]: @jw642459986, attributes for leftButton span more than one line. The closing bracket should be on a new line.

File: client/src/index.jsx [hunk 2]
```diff
@@ -78,17 +79,17 @@ class App extends React.Component {
             <div className="BottomCarousel">
                 <h3 className="Module_Header">Similar Products</h3>
                 <Container>
-                    <Left_Button 
+                    <LeftButton 
                     onClick={() => this.handleLeftClick()}
                     disabled={this.state.first === 0}>
-                    </Left_Button>
+                    </LeftButton>
```
Comment (Ahmed-Elawad) [inline client/src/index.jsx:85]: @jw642459986, are you using the AirBnB style guide? LeftButtons attributes span multiple lines so the closing brace should be on it's own line.

File: client/src/index.jsx [hunk 3]
```diff
@@ -78,17 +79,17 @@ class App extends React.Component {
...
                     {this.state.displayed.map((item) => {
                         return (<Carousel_item key={item['_id']} item={item}/>);
                     })}
-                    <Right_Button
+                    <RightButton
                     onClick={() => this.handleRightClick()}
                     disabled={this.state.last === this.state.data.length - 1}>
-                    </Right_Button>
+                    </RightButton>
```
Comment (Ahmed-Elawad) [inline client/src/index.jsx:92]: @jw642459986, Right button attr span multiple lines. Closing brace should be on new line.

**Review by Ahmed-Elawad — COMMENTED** [review]
@jw642459986, The changes look good. A few adjustments are needed in the client/src/index.js file for styling. Attributes that span multiple lines should have a closing brace on a new line.

**Pull request merged**